examples/with-nextjs-and-clerk-auth/src/lib/monite-api/demo-data-generator/counterparts.ts

```diff
@@ -57,10 +67,21 @@ interface ICounterpartsBuilderOptions {
     enabled: boolean;
     count: number;
   };
+
+  contacts: {
+    /**
+     * Create counterparts with contacts
+     *
+     * By default, it is true, and contacts will be created
+     *  for the counterparts
+     */
+    enabled: boolean;
+    count: number;
+  };
 }
 
-interface IConterpartsServiceResponse {
-  counterparts: Array<components['schemas']['CounterpartResponse']>;
+interface ICounterpartsServiceResponse {
```

Comment:
If it's not too much trouble, let's also change the interface names (we had [a proposal](https://monite.atlassian.net/browse/DEV-10062)).

```ts
ICounterpartsServiceResponse => CounterpartsServiceResponse

ICounterpartsServiceResponse => CounterpartsServiceResponse
```

I think the size of PR will not increase significantly.